src/R2.jl

```diff
@@ -20,10 +17,23 @@ export R2, R2Solver
 - max_eval::Int: maximum number of evaluation of the objective function
 - max_time::Float64 = 3600.0: maximum time limit in seconds
 - verbose::Int = 0: if > 0, display iteration details every `verbose` iteration.
-
 # Output
 The value returned is a `GenericExecutionStats`, see `SolverCore.jl`.
-
+# Callback
+The callback is called after each iteration.
+The expected signature of the callback is `(nlp, solver)`, and its output is ignored.
+Notice that changing any of the input arguments will affect the subsequent iterations.
+In particular, setting `solver.output.status = :user` will stop the algorithm.
+All relevant information should be available in `nlp` and `solver`.
+Notably, you can access, and modify, the following:
+- `solver.x`: current iterate.
```

Comment:
```suggestion
- `solver.x`: current iterate;
```
end each item with ";".